### ISSUE TITLE
Search: Inverted Focus State on Clear/Close Buttons in Search Modal

### ISSUE THREAD

**Issue by katiebethbrown** [Automattic/jetpack#23402]
### Impacted plugin

Jetpack

### Steps to Reproduce

The default state on buttons in the search modal uses a style like the :focus style, which is confusing, and the :focus state uses a plain thin dotted outline.

Default State:
<img width="200" alt="CleanShot 2022-03-11 at 17 26 06@2x" src="https://user-images.githubusercontent.com/42279181/157995252-cce24123-a8fd-4938-b8cb-9586213e4f85.png">

Clear Focused:
<img width="200" alt="CleanShot 2022-03-11 at 17 26 29@2x" src="https://user-images.githubusercontent.com/42279181/157995273-2573367d-b051-4d45-b69a-7162a49658a9.png">

Close Focused:
<img width="200" alt="CleanShot 2022-03-11 at 17 26 49@2x" src="https://user-images.githubusercontent.com/42279181/157995283-361ab4ae-054e-40bd-9c7e-afc6e1f5af9f.png">

### Other Information
This issue was surfaced to team 51 by an a11y contractor working on a partner site.

Also noting that the default state may have already changed from the time this was originally surfaced. In the original screenshot (if I'm making sense fof it correctly) it looks like there was a thin dotted outline around the clear button in the default state:

<img width="200" src="https://user-images.githubusercontent.com/1686451/129969779-ba359119-35fc-46ff-9f06-c4cbb967aa56.png">

**Comment by kangzj** [comment]
Hi @katiebethbrown 

Thanks for reporting this 👍 

I think the problem was the close button styles were overridden theme styles, and [the fix](https://github.com/Automattic/jetpack/pull/20065) was shipped a while ago.

So I suppose the problem doesn't exist anymore. If you still have issues, just let us know. Thanks.

Cheers

**Comment by bluefuton** [comment]
I think there's still something to fix here, so reopening. The default styles still have a thin dotted line on focus (shown here in P2):

![Screen Shot 2022-03-25 at 14 42 50](https://user-images.githubusercontent.com/17325/160143130-7d745bde-02a3-45f2-8531-cb3f0b819c34.png)

**Comment by annacmc** [comment]
@bluefuton I might be wrong here, but I think the thin dotted line on focus is okay. Just not as the default state, which it is no longer displaying in. I've been looking into this a bit, and it seems the dotted lines are more often firefox standard, and solid lines are safari. but overall when I check out Accessibility for Everyone is say:

> focus styles (frequently displayed by browsers as a combination of dotted borders and blue glows) are indispensable. 

Which leads me to believe either dotted border or blue glow is OK. 

We just only want them to show on-focus. Which, if I am interpreting correctly, we are. I do see that these two elements use the dotted borders, and our other overlay elements use the blue glows. Is that the issue here, that they are different? Or is there something else I'm missing?

**Comment by bluefuton** [comment]
> Which, if I am interpreting correctly, we are. I do see that these two elements use the dotted borders, and our other overlay elements use the blue glows. Is that the issue here, that they are different? 

@annacmc yes, I think it was the lack of consistency I was highlighting. All the other elements on the page use the blue solid border, but the close button uses the dotted outline. It'd be good if we could update the close button to behave the same way.
